R/crunchy.R

```diff
@@ -0,0 +1,23 @@
+#' @importFrom shiny div includeCSS includeScript tags
+.onLoad <- function (lib, pkgname="crunchy") {
```

Comment:
I think the answer to this is no but to confirm, will this cause problems if `library(crunchy)` is called multiple times?

Reply:
I'm fairly certain that `package::.onLoad` is only called when a package's namespace is first loaded. Subsequent `library` or `crunchy::whatever` calls wouldn't re-load the package.

Even if it did, if you call `trace` more than once, the last tracer wins--they replace any tracer that was already there. So there would be no harm to calling this many times.